pages/cart-page/cart-page.css

```diff
@@ -0,0 +1,60 @@
+.cart-section {
+    width: 70%;
+    margin: auto;
+    display: grid;
+    grid-template-columns: 1.5fr 1fr;
+    gap: 2rem;
+}
+.cart-section > * {
+    align-self: flex-start;
+    justify-self: center;
+    width: 100%;
+}
+.cart-section .card {
+    margin: 0;
+}
+
+/* card quantity */
+
+.cart-section .card .card-quantity {
+    width: 100%;
+    margin: 0;
+}
+.cart-section .card-quantity input {
+    width: 2rem;
+    font-weight: var(--fw-bold-600);
+    text-align: center;
+    padding: var(--padding-xs);
+    margin: 0;
+}
+
+.card-quantity input[type="number"]::-webkit-inner-spin-button,
+.card-quantity input[type="number"]::-webkit-outer-spin-button {
+    -webkit-appearance: none;
+    -moz-appearance: none;
+}
+
+/* cart summary */
+.cart-summary {
+    width: 100%;
+    max-width: 32rem;
+    border: 1px solid var(--clr-primary-faded);
+    border-radius: 4px;
+    padding: var(--padding-md);
+    box-shadow: var(--box-shadow-light);
+}
+.cart-summary > * {
+    margin-bottom: var(--margin-xs);
+}
+.cart-summary i {
+    margin-right: var(--margin-xs);
+}
+.cart-summary ~ .btn {
+    width: 100%;
+    margin: var(--margin-sm) 0;
+}
+.apply-coup-btn {
+    width: max-content;
+    margin: 0;
```

Comment:
You are using `margin:0;` in multiple places maybe make a utility class for it.

Reply:
I'm resetting the margins that I applied in the component library, once I'm done with ecomm, I'll update thra margins in CL and temove it here.

Thanks for the feedback